permiso/src/main/java/com/greysonparrelli/permiso/PermisoDialogFragment.java

```diff
@@ -129,4 +163,47 @@ public void setOnCloseListener(IOnCloseListener listener) {
          */
         void onClose();
     }
+
+    public static class Builder {
+        private int titleId = 0;
+        private String title = null;
+        private int buttonTextId = 0;
+        private String message = null;
+        private int messageId = 0;
+        // by default, interpret the dialog msg body as string text
+        private boolean interpretHtml = false;
+        private String buttonText = null;
+
+
+        public Builder() {}
+
+        public Builder(int titleId, int msgBody, int buttonTextId) {
+            this();
```

Comment:
Don't see a need for this. If you want to guard against code changes in one constructor becoming inconsistent with the other constructors, I'd actually do this in reverse. Have the no-arg constructor just call ```this(0, 0, 0)```. However, I don't think that's necessary. Up to you!